### DISCUSSION TITLE
Subclassing Guild

### DISCUSSION THREAD

**Discussion by AllegroVivo** [Pycord-Development/pycord#1439] [Q&A] [answered]
Hey folks. For the bot I'm working on I want to subclass and add a few attributes to the Guild class. Is there a function I need to overwrite somewhere that will allow me to pass in the subclassed Guild?

I know there's an example in the repository for passing in a custom Context, so I figured this should be possible as well.

**Comment by Lulalaby** [comment, marked as answer]
Didn't you asked that on our discord server?

**Reply by AllegroVivo** [reply]
Yis, someone recommended I ask here too just to cover all my bases. I hope that's okay? >.<

**Reply by Lulalaby** [reply]
We prefer such questions on our guild if you're a member. 

GitHub is more a place for feature requests and bug reports. 
And the discussion page idk kinda useless.

**Reply by AllegroVivo** [reply]
Ah. Sorry.
